client/cmd/dexcctl/simnet-setup.sh

```diff
@@ -41,7 +41,7 @@ if [ $ETH_ON -eq 0 ]; then
 fi
 
 echo registering with DEX
-./dexcctl -p abc --simnet register 127.0.0.1:17273 100000000 ~/dextest/dcrdex/rpc.cert
+./dexcctl -p abc --simnet register 127.0.0.1:17273 100000000 42 ~/dextest/dcrdex/rpc.cert
```

Comment:
Will have to take this out if going ahead of btc-fees PR.  I snagged this change for it anyway.

Reply:
ah